src/main/resources/assets/paths/mailByRange.yaml

```diff
@@ -0,0 +1,49 @@
+get:
+  summary: Get Mail Messages by Date Range.
+  description: |
+    Admin only feature returns MailMessage list filtered by Date Range.
+  parameters:
+    - name: start
+      in: query
+      description: | 
+        Earliest Date to search in MM/dd/yyyy format. (e.g. 05/19/2019)
+      required: true
+      schema:
+        type: string
+    - name: end
+      in: query
+      description: |
+        Latest Date to search in MM/dd/yyyy format. (e.g. 05/20/2022). Defaults to the future if empty.
+      required: false
+      schema:
+        type: string
+    - name: limit
+      in: query
+      description: Number of results to return.  Defaults to 20 if unset. May not be negative
+      required: false
+      schema:
+        type: integer
+    - name: offset
```

Comment:
feedback from before, but i am just curious on thoughts: any interest in an optional parameter to filter by type as well? i feel usually you would be debugging / trying to find specific emails of a known type when searching for date as well.

Reply:
Filtering on type is a different use case that we don't actually have yet. I can foresee the need for it, but I would prefer that we build that feature when we actually need it.